lib/reply.js

```diff
@@ -387,12 +387,20 @@ function onSendEnd (reply, payload) {
 
 function sendStream (payload, res, reply) {
   var sourceOpen = true
+  var errorLogged = false
 
   eos(payload, { readable: true, writable: false }, function (err) {
     sourceOpen = false
     if (err != null) {
       if (res.headersSent) {
-        reply.log.warn({ err }, 'response terminated with an error with headers already sent')
+        if (!errorLogged) {
+          errorLogged = true
+          if (err.code === 'ERR_STREAM_PREMATURE_CLOSE') {
+            reply.log.info({ res }, 'stream closed prematurely')
```

Comment:
Do we really need to log this? The server shouldn't really care, right? Maybe log at `debug` or `trace`? I'm not sure.

Reply:
Sending a normal response is currently logged as 'request complete' (INFO).
Closing the stream prematurely is the equivalent I think.
If we don't log that you would have requests that have a log line for when the request came but nothing else and thus no way of knowing what happened to the request.

Reply:
In that case, this should honor the request logging disablement feature.

Reply:
I can't find reference/docs for that feature. Can you point me to it?

Reply:
https://www.fastify.io/docs/latest/Server/#disablerequestlogging